samples/javascript-azurefunction/README.md

```diff
@@ -71,8 +71,14 @@ func extensions install -p Dapr.AzureFunctions.Extension -v <version>
 
 Run function host with Dapr: 
 
+Windows
 ```
-dapr run --app-id functionapp --app-port 3001 --dapr-http-port 3501  --components-path ..\components\ -- func host start --no-build
+dapr run --app-id functionapp --app-port 3001 --dapr-http-port 3501  --components-path ..\components\ -- func host start
```

Comment:
Is it ok to remove the --no-build option from here? I hope so

Reply:
It does not matter....so let me remove it